awscli/customizations/wizard/wizards/iam/_new-role-wip.yml

```diff
@@ -107,5 +109,49 @@ plan:
         condition:
           variable: wants_config_profile
           equals: yes
-#execute:
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+#      Just a random text in templates to check the functionality
+      preview_none:
+        type: template
+        value: ''
+      preview_cli_command:
+        type: template
+        value: 'aws command'
+      preview_cfn_template:
+        type: template
+        value: |
+          {{
+            "Version": "2008-10-17",
+            "Statement": [
+                {{
+                    "Action": "sts:AssumeRole",
+                    "Principal": {{
+                        "Service": "{service_name}.amazonaws.com"
+                    }},
+                    {% if wants_config_profile %}
+                    "Yes!!!! I really want a config profile"
+                    {% endif %}
+                    "Effect": "Allow",
+                    "Sid": ""
+                }}
+            ]
+          }}
+      output_type:
+        type: prompt
+        description: elect an execution preview format
+        choices:
+          - display: None
+            actual_value: preview_none
+          - display: CLI command
+            actual_value: preview_cli_command
+          - display: CloudFormation template
```

Comment:
Nit pick let's add AWS to the beginning to the display values to make it consistent with other references to products in the wizard.